packages/playground/Samples/textinput.tsx

```diff
@@ -120,6 +120,11 @@ export default class Bootstrap extends React.Component<{}, any> {
             maxLength={10}
             keyboardType="numeric"
           />
+          <TextInput
```

Comment:
You should add this as an example to TextInputExample.windows.js - And add a new test to the E2E test app to verify that it works.